Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.6-alpine
+
+RUN mkdir -p /go/src/redoctober
+WORKDIR /go/src/redoctober
+
+# https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache
+RUN apk --no-cache add git openssl
+
+# build binary
+COPY . /go/src/redoctober
+RUN go get -d -v
+RUN go install -v
+
+# cleanup
+RUN apk del git
+RUN rm -rf /var/cache/apk/*
```

Comment:
Would it be fair to say that having multiple `RUN`s optimizes re-build times, whereas having them all combined into one `RUN` optimizes the download size of the image? Totally fine with switching it—just want to understand the rationale.